identity/src/main/java/com/android/identity/GattClient.java

```diff
@@ -448,6 +587,11 @@ void sendMessage(@NonNull byte[] data) {
         drainWritingQueue();
     }
 
+    // When using L2CAP it doesn't support characteristics notification
+    public boolean supportsTransportSpecificTerminationMessage() {
+        return !mSupportL2CAP;
+    }
```

Comment:
I think the standard actually allows for this. Our code should probably also allow for it.